pages/ReportsList/index.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import Button from "../../components/Button";
+import Forms from "../../components/ReportsLists";
+import { WorkReports } from "../../DB/data";
+import useLocalStorageState from "use-local-storage-state";
+import { useRouter } from "next/router";
+
+const StyledSection = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export default function ReportsList() {
+  const router = useRouter();
+  const [newWorkReports, _] = useLocalStorageState("workReport", {
```

Comment:
So you are not using the localStorage to save any information? Why implementing it at all then?